index.js

```diff
@@ -1,57 +1,115 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import fetch from 'node-fetch';
+import weaviate from 'weaviate-client';
 
 const app = express();
 
-app.use(bodyParser.json())
+app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
-const URI = 'http://docker40937-typhon-server.th1.proen.cloud'
-const INDEX = 'poster'
+const URI = 'http://docker40937-typhon-server.th1.proen.cloud';
```

Comment:
I think you should keep it as credential information in .env file or export as user data on your cloud server